test/contrib/test_gae_manager.py

```diff
@@ -57,9 +61,6 @@ def _absolute_url(self, path):
 class TestGAEConnectionManager(TestConnectionPool):
     __test__ = True
 
-    # Magic class variable that tells NoseGAE to enable the URLFetch stub.
-    nosegae_urlfetch = True
```

Comment:
Okay I think I'm going to switch from pytest_gae which looks like a unmaintained project (?) to pytest_beds which at least mentions urlfetch in it's docs. Will do some more research.